pints/_log_likelihoods.py

```diff
@@ -465,6 +465,90 @@ def evaluateS1(self, parameters):
         return L, dL
 
 
+class GaussianIntegratedLogUniformLogLikelihood(pints.ProblemLogLikelihood):
```

Comment:
Add a note (quite early on) saying how this is different from the GaussianIntegratedUniformLogLikelihood

Reply:
OR, since the code for this class seems to be simpler, do it the other way around